ironfish-rust/src/test_util.rs

```diff
@@ -60,17 +60,10 @@ pub(crate) fn auth_path_to_root_hash(
     cur
 }
 
-// Helper function to create a list of random identifiers for multisig participants.
-pub fn create_identifiers(num_identifiers: usize) -> Vec<Identifier> {
-    let mut identifiers = Vec::new();
-
-    for _ in 0..num_identifiers {
-        identifiers.push(
-            Secret::random(thread_rng())
-                .to_identity()
-                .to_frost_identifier(),
-        );
-    }
-
-    identifiers
+/// Helper function to create a list of random identifiers for multisig participants.
+pub fn create_multisig_identifiers(num_identifiers: usize) -> Vec<Identifier> {
```

Comment:
rename `create_multisig_identities`

Also should the return type here be `Vec<Identity>`?

Reply:
Oh cool, I see the collect is so that you can gfet back the `Identifiers`